### ISSUE TITLE
"no-duplicate-selectors" rule matches in @keyframes

### ISSUE THREAD

**Issue by morishitter** [stylelint/stylelint#880]
This is my code:

```css
@keyframes toast-in {
  0% {
    opacity: 0;
    transform: translateY(-16px);
  }

  100% {
    opacity: 1;
    transform: none;
  }
}

@keyframes toast-out {
  0% {
    opacity: 1;
    transform: none;
  }

  100% {
    opacity: 0;
    transform: translateY(-16px);
  }
}
```

If we set `"no-duplicate-selectors": true` in stylelint config, occurs the following warning message. (using stylelint v4.5.0)

```
⚠  Unexpected duplicate selector "0%" (no-duplicate-selectors) [stylelint]
⚠  Unexpected duplicate selector "100%" (no-duplicate-selectors) [stylelint]
```

Are these expected result?

I propose that rules in `@keyframes` do not have to match "no-duplicate-selectors" rule.

Thanks.

**Comment by jeddy3** [comment]
> Are these expected result?

Nope.

> I propose that rules in @keyframes do not have to match "no-duplicate-selectors" rule.

Agreed. Fixing now.

**Comment by jeddy3** [comment]
@morishitter FYI, released in `4.5.1`

**Comment by morishitter** [comment]
@jeddy3 Thanks. Nice work :)
